packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -16,14 +16,18 @@ import './index.scss';
 import { config } from '../../lib/config';
 
 export type NewUserEmailFormProps = {
-  getString: (id: string) => string;
-  checkAccountExists: Function;
+  getString?: (id: string) => string;
+  setValidEmail: Dispatch<SetStateAction<string>>;
+  setAccountExists: Dispatch<SetStateAction<boolean>>;
```

Comment:
nit: Those are definitely the types for those functions.  But they are harder for other engineers to read.  `(value: string) => void;` and `(value: boolean) => void;` make it obvious what the param and return types are.